config/initializers/sac.rb

```diff
@@ -0,0 +1,6 @@
+SAC_CATEGORIES = [
```

Comment:
Configuration like this should live in the `config` directory.
I think `config/application.rb` is the best location, as comparable configuration also lives there.
There currently is no way to edit the categories as an admin, but since this won't change too often, it is fine.
Make sure the categories are filled with something sensible before merging though, as this configuration will be used in production.
@Jelte-Akker should be able to help you out with the relevant categories and points.

Reply:
Apparently the categories and points are linked in the issue, so you can use that.